src/link_prober/IcmpPayload.h

```diff
@@ -130,6 +184,10 @@ struct IcmpPayload {
 static_assert(sizeof(IcmpPayload) % 2 == 0,
               "ICMP Payload size should be even sized, please add zero padding");
 
+static_assert(sizeof(TlvCommand) % 2 == 0, "TLV size should be even sized, please add zero padding");
```

Comment:
Why `TlvSentinel` and `TlvCommand` must be even sized? We may combine several TLC struct in a ICMP payload.

Reply:
Now we assume all TLVs are even-sized so it is more easier to calculate the checksum